data/hai/culture hai.txt

```diff
@@ -0,0 +1,208 @@
+				decline
+			label translator
+			`	As you watch the triangle elder being holoprojected seems to flicker and then fizzles out as the battery unit fades, apparently under the assault of the root vegetable which appears to do a victory dance. To your surprise your translator pipes up, "[Victory, Success] for the true and square root that is more [significant, meaningful] than the tripartite [system, tradition] of the elders!"`
+			`	A little further on you encounter two different groups of protesters weaving in and out of the spaceport's entrances like some kind of conga line. The triangular elder and root vegetable make several reappearances on their placards, some of them holographically animated.`
+			`	A different security guard, this one in regular attire, appears at your elbow. "You look a little lost sir but don't worry, it's just a meme war. Carry on with your business and don't touch the meme items, no one should bother you over it."`
+			choice
+				`	"Sure, will do."`
+					goto done
+				`	Your elders have a tripartite system?`
+			`	The security guard chuckles and says, "Oh you can understand? They don't really, but it's a very old joke from an old campaign that there are only 3 tenets the elders care about, and they're all the ones that affect us the least."`
+			`	When you ask what has triggered this now he says, "Oh, the latest education priorities came out with a 4 tenet focus, on the same day that an elder accidentally quoted the old campaign. It has been very entertaining."`
+			choice
+				`	"How strange."`
+				`	"Sounds fun."`
+			label done
+			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
+				decline
+
+mission "ZZCU Wrestling Star"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`A famous Hai wrestler is doing a signing. (TBD)`
+				decline
+
+mission "ZZCU Fencing Star"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`A famous Hai fencer is doing a signing. (TBD)`
+				decline
+				
+mission "ZZCU ??? Star"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`A famous Hai sportsperson of an obscure alien sport is doing a signing. (TBD)`
+				decline
+
+mission "ZZCU Gambling Judgment"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Upon entering a diner at the spaceport you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`
```

Comment:
"Upon entering" and "you have not been seated long" directly conflict.
I was not aware "near as you can tell" was a thing, use "as far" instead possibly.

Reply:
Re-done the first part.
R.E. "as far" -> I have a little bit of a personal crusade against simplifying language for the sake of simplifying language in a _text based_ game. So if the phrasing is legitimate and the meaning is clear, I'm in favour of keeping it.